packages/calcite-components/src/components/block/block.tsx

```diff
@@ -94,6 +94,11 @@ export class Block
    */
   @Prop({ reflect: true }) dragHandle = false;
 
+  /**
+   * Defines the available placements that can be used when a flip occurs.
```

Comment:
Do we want a bit more context here for "a flip occurs" - maybe something about the rendered action menu flipping? It makes sense to me as someone who knows it refers to internally rendered action menu, but maybe these could be confusing without a more descriptive description (or even prop name)? Applies to others.

Reply:
Yeah probably. I copied this from another component.

cc @geospatialem @DitwanP on doc

Reply:
I guess my comment was more about _what_ is being affected by this. In this case, it's "the action menu that is rendered when you slot actions in ___"

"... the component's fallback `placement`" still isn't clear to a user without context - maybe "Specifies the fallback `placement` of the component's (internally rendered Action Menu that displays when slotted actions overflow)" (but ... less bad verbiage)

Reply:
@geospatialem combobox is out of scope I think. It already has flipPlacements just not placement.

Popover, input-date-picker, split-button, and dropdown already has these properties.

Reply:
Added a new PR https://github.com/Esri/calcite-design-system/pull/10261 to cover the remaining components for consistency.

Reply:
For consistency and readability, would we want to consider updating similar properties to use the `menu` prefix? I don't recall if we have properties other than `overlayPositioning`? 🤔